src/ReactiveUI.Blazor/ReactiveLayoutComponentBase.cs

```diff
@@ -33,21 +33,21 @@ public class ReactiveLayoutComponentBase<T> : LayoutComponentBase, IViewFor<T>,
         /// </summary>
         public ReactiveLayoutComponentBase()
         {
-            this.WhenAnyValue(x => x.ViewModel).Subscribe(_ => StateHasChanged());
-            var viewModelsPropertyChanged = this.WhenAnyValue(x => x.ViewModel)
-                .Where(x => x != null)
-                .Select(x => Observable.FromEvent<PropertyChangedEventHandler, Unit>(
-                    eventHandler =>
-                    {
-                        void Handler(object sender, PropertyChangedEventArgs e) => eventHandler(Unit.Default);
-
-                        return Handler;
-                    },
-                    eh => x.PropertyChanged += eh,
-                    eh => x.PropertyChanged -= eh))
-                .Switch();
-
-            viewModelsPropertyChanged.Do(_ => StateHasChanged()).Subscribe();
+            var viewModelsChangedObservable = this.WhenAnyValue(x => x.ViewModel)
+            .Where(x => x != null)
```

Comment:
Change back to indented in both files thanks

Reply:
Done